### PR TITLE
Add OPENJCEPLUS_SUPPORT flag in JPP tags

### PR DESCRIPTION
**Pull request by taoliult**
When building the OpenJCEPlus with Semeru OpenJDK, some java.base packages need to be "exports" to the OpenJCEPlus module. But not all the platforms are supported by OpenJCEPlus. For those not supported platforms, export to OpenJCEPlus module will cause the unknown module error. So, by adding the OPENJCEPLUS_SUPPORT flag in JPP Tags, it will only allow the OpenJCEPlus required java.base packages to be exported on the supported platforms, to avoid the unknown module error.

### CODE REVIEW

File: jcl/src/java.base/share/classes/module-info.java.extra [hunk 1]
```diff
@@ -78,3 +81,21 @@ opens jdk.internal.misc to
 opens java.lang to
     openj9.criu;
 /*[ENDIF] CRIU_SUPPORT */
+/*[IF OPENJCEPLUS_SUPPORT]*/
+exports jdk.internal.logger to
+    openjceplus;
+/*[IF !CRIU_SUPPORT]*/
+exports jdk.internal.misc to
+    openjceplus;
```
Comment (pshipton) [inline jcl/src/java.base/share/classes/module-info.java.extra:89]: This was already exported on line 72

Comment (taoliult) [reply jcl/src/java.base/share/classes/module-info.java.extra:89]: @pshipton 
I added the check here, is for, when the CRIU is not supported, then the line 72 will not be added into the module-info.java file. So, I added a check here, if CRIU is not supported, then add the `exports jdk.internal.misc to  openjceplus;` here for OpenJCEPlus.

Comment (pshipton) [reply jcl/src/java.base/share/classes/module-info.java.extra:89]: I'm not 100% sure that will work, but we can try if you haven't already.

Comment (pshipton) [reply jcl/src/java.base/share/classes/module-info.java.extra:89]: Anyway if we move all this to the extensions then it doesn't matter.

Comment (taoliult) [reply jcl/src/java.base/share/classes/module-info.java.extra]: Moved all these to the extensions module-info.java.

**Comment by pshipton** [comment]
You still need to sign the ECA before this can be accepted, or I could make this simple change instead.

**Review by keithc-ca — APPROVED** [review]

**Pull request merged**